workspaces/src/network/testnet.rs

```diff
@@ -88,6 +88,7 @@ impl TopLevelAccountCreator for Testnet {
                     logs: Vec::new(),
                     receipt_ids: Vec::new(),
                     gas_burnt: 0,
+                    tokens_burnt: 0,
                     executor_id: "testnet".parse()?,
                     status: ExecutionStatusView::SuccessValue(String::new()),
                 },
```

Comment:
yup, certainly not ideal. This was just mocked out at the time to get testnet going, but let's revisit it in a future issue/PR since it's going to require finding the actual transaction details themselves which the Helper Contract AFAIK does not provide additional details for when creating an account